examples/mysql-galera/pxc-node1.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: Service
+id: pxc-node1
+metadata:
+  name: pxc-node1
+spec:
+  ports:
+    - port: 3306
+      targetPort: 3306
+      name: mysql
+    - port: 4444
+      targetPort: 4444 
+      name: state-snapshot-transfer
+    - port: 4567
+      targetPort: 4567
+      name: replication-traffic 
+    - port: 4568
+      targetPort: 4568 
+      name: incremental-state-transfer 
+  selector:
+    name: pxc-node1 
+labels: 
+  name: pxc-node1
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: pxc-node1 
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: pxc-node1 
+        unit: pxc-cluster
+    spec:
+      containers:
+        - resources:
+            limits: 
+              cpu: 0.3
+          image: capttofu/percona_xtradb_cluster_5_6:beta
+          name: pxc-node1
+          ports:
+            - containerPort: 3306
+              name: mysql
```

Comment:
I did that because it actually requires legitimate names (RFC6335) and there are no service names for the other ports. I did as you suggested as it doesn't really do anything. For the overall cluster service, I used it since that's the only port  for that service.